#goal definition
bool pushDoor
---
#result definition
bool success
---
#feedback
bool success
